.github/workflows/ci-actions.yml

```diff
@@ -0,0 +1,249 @@
+name: CI
+
+on:
+  push:
+    paths-ignore:
+      - "README.md"
+  pull_request:
+    types: [assigned, opened, synchronize, reopened, ready_for_review, edited]
+    paths-ignore:
+      - "README.md"
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
Is this meant to create a nightly build?
We already have one so it's not mandatory to have it here as well and keep sonatype as the snapshot versions repo

Reply:
removed